common-lib/src/main/scala/com/gu/mediaservice/lib/auth/Store.scala

```diff
@@ -68,9 +68,20 @@ object PermissionType extends Enumeration {
   val EditMetadata = Value("editMetadata")
   val DeleteImage  = Value("deleteImage")
   val DeleteCrops  = Value("deleteCrops")
+  val BigSpender   = Value("bigSpender")
```

Comment:
might want to add this key to the [sample permissions file](https://github.com/guardian/grid/blob/master/cloud-formation/scripts/permissions.properties) too